zulipterminal/ui_tools/views.py

```diff
@@ -1113,7 +1114,7 @@ def __init__(
             else []
         )
         contents = [
-            ("Application", [("Zulip Terminal", zt_version)]),
+            ("Application", [("Zulip Terminal", zt_version), ("Platform", PLATFORM)]),
```

Comment:
This looks OK, though I had originally envisaged it being in a separate section below the others, rather than in the `Application` section.

Reply:
I assumed the version and the platform would go together? I'll put it in a separate section, that does sound better.